modules/snowflake_connector_config/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "connector_name" {
+  type = string
+  description = "Name for the Kafka Connect connector.  Use kabab-case."
+}
+
+variable "kafka_connect_url" {
+  description = "URL, including https://, of Kafka Connect to run the connector on"
+  type = string
+}
+
+variable "kafka_topic" {
+  type        = string
+  description = "Comma-separated list of topics to sync to Snowflake"
+}
+
+variable "snowflake_url" {
+  type        = string
+  description = "The full URL for accessing Snowflake.  Protocol and port are optional"
+}
+
+variable "snowflake_username" {
+  type        = string
+  description = "Username to use with this Snowflake connector"
+}
+
+variable "snowflake_private_key" {
```

Comment:
I suppose we should treat the private key the same as a password (i.e. it shouldn't be in the code repository).

So this should be changed to a `data "aws_ssm_parameter"`